### ISSUE TITLE
There is no requirements.txt file

### ISSUE THREAD

**Issue by ferid333** [SriTejaa/Email_GUI#1]
There is no requirements.txt file in repo

**Comment by SriTejaa** [comment]
Hey i ferid,
I don't get your point actually correct me if i'm wrong - are you asking about the prerequisites for the program?

**Comment by ferid333** [comment]
Yes, you should add requirements.txt file in repo so users can install dependencies before using the code

**Comment by ferid333** [comment]
I can do it if you want
